CHANGES.rst

```diff
@@ -341,7 +341,7 @@ accordingly to an ``EBLAbsorptionNormSpectralModel``. A new
 have been introduced.
 
 Gammapy v0.18 comes now with support for custom energy dependent spatial models.
-An example for this can be found in the `models tutorial <tutorials/models.html>`__.
```

Comment:
Is it possible to fix the links instead of removing it?

Reply:
Ok, done.
I have replaced the links with absolute links to the versioned docs, instead of removing.